src/actions/index.js

```diff
@@ -1,29 +1,86 @@
 import Axios from 'axios';
 import YTSearch from 'youtube-api-search';
 
-const API_KEY = 'AIzaSyCQOrCyZMNUT9PwPes5AJUyn1kopJm2xU8';
-const YOUTUBE_URL = 'https://www.googleapis.com/youtube/v3/search';
 export const FETCH_VIDEOS = 'FETCH_VIDEOS';
+export const FETCH_PLAYLIST = 'FETCH_PLAYLIST';
 export const SELECT_VIDEO = 'SELECT_VIDEO';
+export const SEARCH_METHOD = 'SEARCH_METHOD';
+export const CATEGORY_SEARCH = 'CATEGORY_SEARCH';
+export const CATEGORY_VALUE = 'CATEGORY_VALUE';
+const API_KEY = 'AIzaSyCQOrCyZMNUT9PwPes5AJUyn1kopJm2xU8';
+const YOUTUBE_URL = 'https://www.googleapis.com/youtube/v3/search';
+const YOUTUBE_URL_PLIST = 'https://developers.google.com/apis-explorer/#p/youtube/v3/youtube.playlists.list'
+
+export function fetchVideos(keyword, searchMethod, categorySearch, categoryValue) {
+  var params;
+  if (searchMethod && searchMethod === "short") {
```

Comment:
is there any need for these if else constructions ??? 

params = {
   part: 'snippet',
   key: API_KEY,
   q: keyword,
   videoDuration: searchMethod, // maybe the name should be changed
   type: 'video'
}